frontend/app/src/components/home/P2PSwapContent.svelte

```diff
@@ -0,0 +1,295 @@
+                    }
+                });
+        }
+
+        return Promise.resolve();
+    }
+
+    function accept() {
+        confirming = false;
+
+        if (me) {
+            client
+                .acceptP2PSwap(
+                    messageContext.chatId,
+                    messageContext.threadRootMessageIndex,
+                    messageId,
+                )
+                .then((resp) => {
+                    if (resp.kind !== "success") {
+                        toastStore.showFailureToast(i18nKey("p2pSwap." + resp.kind));
+                    }
+                });
+        }
+    }
+</script>
+
+{#if confirming}
+    {#if me}
+        <AreYouSure
+            message={i18nKey("p2pSwap.confirmCancel", {
+                amount: fromAmount,
+                token: content.token0.symbol,
+            })}
+            action={cancel} />
+    {:else}
+        <AcceptP2PSwapModal
+            ledger0={content.token0.ledger}
+            ledger1={content.token1.ledger}
+            amount0={content.token0Amount}
+            amount1={content.token1Amount}
+            on:accept={accept}
+            on:close={() => (confirming = false)} />
+    {/if}
+{/if}
+
+<div class="swap">
+    <div class="top">
+        {#if content.status.kind === "p2p_swap_open"}
+            <div class="countdown" class:rtl={$rtlStore}>
+                <Clock size={"1em"} color={"#ffffff"} />
+                <span>{timeRemaining}</span>
+            </div>
+        {/if}
+        <div class="coins">
+            <div class="coin">
+                <SpinningToken logo={fromDetails.logo} spin={false} />
+                <div class="amount">{fromAmount} {content.token0.symbol}</div>
+            </div>
+
+            <div class="swap-icon"><SwapIcon size={"2.5em"} /></div>
+
+            <div class="coin">
+                <SpinningToken logo={toDetails.logo} spin={false} />
+                <div class="amount">{toAmount} {content.token1.symbol}</div>
+            </div>
+        </div>
+    </div>
+    <div class="bottom">
+        {#if content.caption !== undefined}
+            <div class="caption">
+                {content.caption}
+            </div>
+        {/if}
+        <div class="summary">{summaryText}</div>
+        {#if instructionText !== undefined}
+            <div class="instructions">
+                <Markdown text={instructionText} />
+            </div>
+        {/if}
+        <div class="accept">
+            <ButtonGroup align="fill">
+                <Button
+                    loading={content.status.kind === "p2p_swap_reserved" ||
+                        content.status.kind === "p2p_swap_accepted"}
+                    disabled={buttonDisabled}
+                    hollow
+                    on:click={onAcceptOrCancel}>{buttonText}</Button>
+            </ButtonGroup>
+        </div>
+    </div>
+</div>
+
+<style lang="scss">
+    $accent: var(--prize);
+
+    :global(.accept button) {
```

Comment:
I you _really_ need to use `:global` css you need to be as sure as possible that it's not going to clash with any other global css and that probably means using more specifically names classes than `.accept`. Even if it doesn't clash with anything now it's a bit of an accident waiting to happen.